var/spack/repos/builtin/packages/libxcb/package.py

```diff
@@ -15,12 +16,14 @@ class Libxcb(Package):
     depends_on("xcb-proto")
     depends_on("pkg-config")
 
-    # depends_on('pthread')    # Ubuntu: apt-get install libpthread-stubs0-dev
-    # depends_on('xau')        # Ubuntu: apt-get install libxau-dev
+    # Ubuntu: apt-get install libpthread-stubs0-dev
+    depends_on('libpthread-stubs')
+    depends_on('libXau')  # Ubuntu: apt-get install libxau-dev
```

Comment:
Agree. Removed already.

On Mon, May 23, 2016 at 3:00 PM, Denis Davydov <notifications@github.com>
wrote:

> In var/spack/repos/builtin/packages/libxcb/package.py
> <https://github.com/LLNL/spack/pull/820#discussion_r64266973>:
>
> > @@ -15,12 +16,14 @@ class Libxcb(Package):
> >      depends_on("xcb-proto")
> >      depends_on("pkg-config")
> >
> > -    # depends_on('pthread')    # Ubuntu: apt-get install libpthread-stubs0-dev
> > -    # depends_on('xau')        # Ubuntu: apt-get install libxau-dev
> > +    # Ubuntu: apt-get install libpthread-stubs0-dev
> > +    depends_on('libpthread-stubs')
> > +    depends_on('libXau')  # Ubuntu: apt-get install libxau-dev
>
> same here.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/LLNL/spack/pull/820/files/c222eb0166213bc9da0a8fd43ae42fff61868cdb#r64266973>
>



-- 
BEST Regards,

--
Dhanannjay (D'Jay) Deo

[ नासतो विद्यते भावो नाभावो विद्यते सतः ।
  Nothing unreal exists, nothing real can be threatened |
  (Bhagawad Geeta 2:16) ]